engine/include/cubos/engine/physics/physics_bundle.hpp

```diff
@@ -20,7 +21,11 @@ namespace cubos::engine
 
         float mass = 1.0F;
         glm::vec3 velocity = {0.0F, 0.0F, 0.0F};
+        glm::vec3 angularVelocity = {0.0F, 0.0F, 0.0F};
         glm::vec3 force = {0.0F, 0.0F, 0.0F};
+        glm::vec3 torque = {0.0F, 0.0F, 0.0F};
         glm::vec3 impulse = {0.0F, 0.0F, 0.0F};
+        glm::vec3 angularImpulse = {0.0F, 0.0F, 0.0F};
+        glm::mat3 inertiaTensor = glm::mat3(0.0F);
```

Comment:
Btw `glm::mat3(0.0F)` isn't an identity matrix, it is a matrix filled with zeroes. `glm::mat3(1.0F)` is an identity matrix. I don't understand the math though, so you might have written this on purpose. Pointing this out just to be sure

Reply:
Yup, I know, this is essencially just a default to know when we want to automatically calculate the inertia tensor, bc we're not supposed to have objects with mass 0 and inertia 0, which is what a matrix of all 0s means in this case, when I unpack the physicsBundle I check to see if it's still 0 or it has been replaced, bc if the dev put a specific value then we probably don't want to replace it. Maybe we should make it more explicit, so there isn't any confusion?

Reply:
I would just add a comment explaining that it is only a temporary value, otherwise LGTM!